micro-infra-spring-base/src/main/groovy/com/ofg/infrastructure/web/resttemplate/fluent/HTTPAuthorizationUtils.groovy

```diff
@@ -0,0 +1,20 @@
+package com.ofg.infrastructure.web.resttemplate.fluent
+
+import org.apache.commons.codec.binary.Base64
+
+class HTTPAuthorizationUtils {
+
+    /**
+     * Encodes credentials for HTTP basic authentication
+     * @param username
+     * @param password
+     * @return
+     */
+    public static String encodeCredentials(String username, String password) {
+        String plainCreds = username + ":" + password
+        byte[] plainCredsBytes = plainCreds.getBytes("US-ASCII")
```

Comment:
Use `java.nio.charset.StandardCharsets#US_ASCII` instead of `"US-ASCII"` string constant.